scss/_settings_colors.scss

```diff
@@ -108,6 +109,22 @@ $colors--dark-theme--border-default: rgba($colors--dark-theme--text-default, 0.2
 $colors--dark-theme--border-high-contrast: rgba($colors--dark-theme--text-default, 0.4) !default;
 $colors--dark-theme--border-low-contrast: rgba($colors--dark-theme--text-default, 0.1) !default;
 
+$colors-light-theme--tinted-backgrounds: (
```

Comment:
If they are light theme, we should have equivalent in dark theme.

Also, please move them up next to light theme colors.

But, right now they feel local to Chips, could we move them there for now? Until we need them elsewhere and when we can and want to make sure we cover them in dark theme as well?

Reply:
This would convey the wrong impression - that these are one off colours for chips only. Ideally we should implement them in p-notification as soon as reasonable, too.

Reply:
Ok. But now we call them theme colors, but they don't have equivalent in dark theme.

Reply:
I want the intention to be clear. There should be, but we do not need to do it here as this pr is to unblock the 3.0 release. Dark themed chips are not s blocker at this point. 

I can leasve a TODO (XXX) comment if you think it will help clarify things?

Reply:
It would be helpful.

The other thing that seems inconsistent is use of the map.
Other theme variables don't do this, and we have multiple variants of some (border: high contrast/low contrast - 2 variables;  backgrounds: default, alt, active, hover, overlay; etc). Maybe these backgrounds should follow this convention as well?

And we already have background theme variables, so maybe just add those there?
`$colors-light-theme--background-positive` `$colors-light-theme--background-negative` etc?

Reply:
as discussed, lack of variable name interpolation in scss makes that impractical.